Jenkinsfile

```diff
@@ -174,14 +174,15 @@ pipeline {
             // Newer one looked up the last known good build of master branch.
             // Pipeline currently does not have access that info.
             sh 'rm -rf testbin/ && mkdir -p testbin'
+            sh 'rm -rf srcpath/fuse'
```

Comment:
Is this still needed?

Reply:
Maybe not but it doesn’t hurt anything.  Just making sure the slate is clean